TAO/TAO_IDL/be/be_visitor_sequence/any_op_cs.cpp

```diff
@@ -189,14 +191,14 @@ be_visitor_sequence_any_op_cs::visit_sequence (be_sequence *node)
       << "/// Copying insertion." << be_nl
       << "void operator<<= (" << be_idt_nl
       << "::CORBA::Any &_tao_any," << be_nl
-      << "const " << node->name () << " &_tao_elem)"
+      << "const ::" << node->name () << " &_tao_elem)"
       << be_uidt_nl
       << "{" << be_idt_nl
-      << "TAO::Any_Dual_Impl_T<" << node->name () << ">::insert_copy ("
+      << "::TAO::Any_Dual_Impl_T< ::" << node->name () << ">::insert_copy ("
```

Comment:
Do we really need `::TAO` here, in the core we do use `TAO::Any_Dual_Impl_T` without problems

Reply:
Is there a downside to it?  I haven't tried this but it seems like a user's IDL could cause a name collision.

Reply:
Not sure, just something I observed. I have restarted my TAOX11 test build to make sure it all works there, we depend on versioned namespaces. Will let you know the results tomorrow, already 9PM here ;-)